lexer/lexer.go

```diff
@@ -90,6 +90,18 @@ func (l *Lexer) NextToken() token.Token {
 		l.skipToNextLine()
 		l.skipWhitespace()
 	}
+	
+	// Check for multi-line comment.
+	// Starts with /* and ends with */.
+	// /* and */ must be the only characters on their line
```

Comment:
Requiring that both `/*` and `*/` are on their own lines isn't acceptable IMO.  It should be easy to address that, though.  Something like this:
```go
for (l.ch == '/' && l.peekChar() == '*') {
	// Advance past the two "/*" characters
	l.readChar()
	l.readChar()

	l.skipPastClosingMultilineComment()
	l.skipWhitespace()
}

func (l *Lexer) skipPastClosingMultlineComment() {
	for !(l.ch == '*' && l.peekChar() == '/') {
		l.readChar()
	}
	l.readChar()
	l.readChar()
}
```

Reply:
Should we raise an error if the closing */ characters were forgotten, or simply ignore it as it would be the end of the file ?

Reply:
I'll simply break out of the loop for now, as it's really easy to change afterwards

Reply:
Ideally, it would return an error the same way the parser returns errors (instead of `panic`).  C, for example, also raises an error when you forget to close the multiline comment.

Reply:
I added a PORYSCRIPT WARNING the same way the parser prints errors.
(Using `log.Printf("PORYSCRIPT WARNING: A comment was not properly closed (at %d:%d). I will still translate the file.", l.lineNumber, l.charNumber)`)
However, I'm unsure if I can get the file causing problems using the current interface, and I'm not versed inough into go to know how to display a stacktrace to display where the error comes from :/

Reply:
It should be an error, not a warning.  The lexer will need to be updated to return an optional error, just like the parser.

Reply:
Done.
I don't think this was the best way to do it, but due to how expectPeek was handled (and especially due to the fact the functions that used it had a specific return value on a specific error), I added an additional return value which is returned if not nil.

Reply:
Oh my, I forgot that introducing this returned error would affect every `nextToken()` etc..  Hmm, I need to think about this some more.

Reply:
I think we'll want to change the internals of the lexer such that it constructs the full list of tokens up-front (which _can_ return an error due to an unterminated multiline comment).  That will allow us to preserve the API of `nextToken()` etc..

Please let me know if you're interested in doing this, or if you'd rather I handle it.

Reply:
Sadly, I don't think I'm familiar enough with go and the codebase to handle such a task, so I'll have to let you handle it ^^'